packaging/bin/pki-create-ca.sh

```diff
@@ -70,7 +70,7 @@ enroll() {
 
 
 	touch "${PKIDIR}/private/${CA_FILE}.pem"
-	chmod o-rwx "${PKIDIR}/private/${CA_FILE}.pem" || die "Cannot set CA permissions"
+	chmod 0644 "${PKIDIR}/private/${CA_FILE}.pem" || die "Cannot set CA permissions"
```

Comment:
Shouldn't it be 0640 based on above line?

Reply:
0640 rather than 0644?  It's a key file.